src/app/explore/explore.component.html

```diff
@@ -88,51 +88,65 @@
-                       the metacmpaign, or processing the campaign data in FE before it gets to here. Could also consider moving the `campaign-card` component out of
-                       components library and into this repo for to make it easier to work on if we're not likely to use it anywhere else.
-
-                       Logic for primary figure amount is particularly confusing - we have to pass null here if the campaign is an emergency match fund campaign
-                       as it doesn't make sense to display any primary figure or secondary figure, but the only way to know in FE at present that it's an emergency campaign
-                       is by the fact that it uses match funds and *isn't* regular giving. If it is regular giving we pass a space to take up space in the layout without being visible.
+                       In practice the `|| campaign.isRegularGiving` should be redundant everywhere except on dev machines,
+                        as any campaign that's part of a regular-giving metacampaign will have parentUsesSharedFunds=true.
                        -->
-              <biggive-campaign-card
-                id="campaign-{{ campaign.id }}"
-                [banner]="campaign.imageUri | optimisedImage: 830 | async"
-                [campaignType]="campaign.isMatched ? 'Match Funded' : null"
-                [campaignTitle]="campaign.title"
-                [donateButtonUrl]="
-                  campaign.isRegularGiving ? '/regular-giving/' + campaign.id : '/donate/' + campaign.id
-                "
-                [donateButtonLabel]="campaign.isRegularGiving ? 'Donate monthly' : 'Donate now'"
-                [moreInfoButtonUrl]="'/campaign/' + campaign.id"
-                [organisationName]="campaign.charity.name"
-                [primaryFigureLabel]="
-                  campaign.isRegularGiving || campaign.parentUsesSharedFunds ? '' : 'Match Funds Remaining'
-                "
-                [primaryFigureAmount]="
-                  this.metaCampaign?.usesSharedFunds || campaign.isRegularGiving || campaign.parentUsesSharedFunds
-                    ? ' '
-                    : (campaign.matchFundsRemaining
-                      | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo)
-                "
-                [progressBarCounter]="campaign.isRegularGiving ? null : getPercentageRaised(campaign)"
-                secondary-figure-label="Amount Raised"
-                [secondaryFigureAmount]="
-                  campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo
-                "
-                [isFutureCampaign]="isInFuture(campaign)"
-                [isPastCampaign]="isInPast(campaign)"
-                [datetime]="getRelevantDateAsStr(campaign)"
-              >
-              </biggive-campaign-card>
+              @if (campaign.parentUsesSharedFunds || campaign.isRegularGiving) {
```

Comment:
I think optionally we could simplify this because the fund setup for Regular Giving defines it as shared. Not a blocker.

```suggestion
              @if (campaign.parentUsesSharedFunds) {
```

Reply:
Right, sorry I missed that in unified view as the delete was more prominent